.teamcity/Gradle_Check/projects/RootProject.kt

```diff
@@ -33,6 +33,8 @@ class RootProject(model: CIBuildModel, functionalTestBucketProvider: FunctionalT
 
     params {
         param("env.GRADLE_ENTERPRISE_ACCESS_KEY", "%ge.gradle.org.access.key%")
+        param("NPM_CONFIG_REGISTRY", "https://repo.gradle.org/gradle/api/npm/npm-temporary/")
```

Comment:
That doesn't change anything, since `env.` is missing in front of the property.

Reply:
:facepalm: Fixed